### PR TITLE
test: add test case to verify correct color application

### PR DESCRIPTION
**Pull request by Ramsonodoi**

### CODE REVIEW

File: src/app/components/navbar/navbar.component.html [hunk 1]
```diff
@@ -1,7 +1,12 @@
 <nav class="bg-blue-900 text-white p-2 flex justify-between items-center fixed top-0 left-0 w-full z-10">
```
Comment (Bilal-Abubakari) [inline src/app/components/navbar/navbar.component.html:1]: Going forward do not use tailwind

File: src/app/shared/changecolor.directive.spec.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { ElementRef } from '@angular/core';
+import { ChangecolorDirective } from './changecolor.directive';
+
+describe('ChangecolorDirective', () => {
+  it('should create an instance', () => {
+    const mockElementRef = new ElementRef(null)
+    const directive = new ChangecolorDirective(mockElementRef);
+    expect(directive).toBeTruthy();
```
Comment (Bilal-Abubakari) [inline src/app/shared/changecolor.directive.spec.ts:8]: Wouldn't like to test if it is actually changing the color?

File: src/app/components/navbar/navbar.component.html (hunk 1 above)
Comment (Ramsonodoi) [reply src/app/components/navbar/navbar.component.html:1]: Okay I wont

File: src/app/shared/changecolor.directive.spec.ts (hunk 2 above)
Comment (Ramsonodoi) [reply src/app/shared/changecolor.directive.spec.ts:8]: Please I will raise another PR for that one had an issues at my end that's why.

**Pull request merged**